### DISCUSSION TITLE
Can Gobigger state space become fixed length

### DISCUSSION THREAD

**Discussion by butterfly2sea** [opendilab/DI-engine#698] [Q&A] [answered]
How does the Gobigger state space in DI-engine change from an indefinite length to a fixed length, or if you want to use a neural network, pass the state to the neural network to obtain the actions in the Gobigger, how to solve the problem of the indefinite length of the input dimension of the neural network？

**Comment by jayyoung0802** [comment, marked as answer]
For features with variable lengths, the common approach is to define a maximum length. Features exceeding this length are truncated, while shorter ones are padded.
